src/App.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
-import './App.css';
+import './styles/styles.scss';
```

Comment:
In all of your js files, we follow a convention of imports from 3rd party libraries (such as your first import from react), then a blank line, then imports from other libraries within your project (you don't have any right now), a blank line, and then local imports like './styles/styles.scss'

Please do this in all of your js files